Frameworks/Foundation/NSDecimalNumber.mm

```diff
@@ -186,214 +207,326 @@ + (NSDecimalNumber*)decimalNumberWithDecimal:(NSDecimal)decimal {
  @Status Stub
  @Notes
 */
-+ (NSDecimalNumber*)decimalNumberWithMantissa:(unsigned long long)mantissa exponent:(short)exponent isNegative:(BOOL)isNegative {
+- (instancetype)initWithString:(NSString*)numericString locale:(id)locale {
     UNIMPLEMENTED();
     return StubReturn();
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
-+ (NSDecimalNumber*)decimalNumberWithString:(NSString*)numericString locale:(id)locale {
-    UNIMPLEMENTED();
-    return StubReturn();
+- (NSDecimal)decimalValue {
+    // Read-Only
+    NSDecimal decimal = { 0 };
+    NSDecimalCopy(&decimal, _decimal);
+    return decimal;
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
-+ (NSDecimalNumber*)one {
-    UNIMPLEMENTED();
-    return StubReturn();
+- (BOOL)isEqual:(NSObject*)objAddr {
+    if ([objAddr isKindOfClass:[NSDecimalNumber class]]) {
```

Comment:
>if ([objAddr isKindOfClass:[NSDecimalNumber class]]) { [](start = 4, length = 54)

An NSDecimalNumber should compare equal to other NSNumbers with the same value, even if they aren't NSDecimalNumbers.